### PR TITLE
feat(blog) Add blog support

### LINKED ISSUE #16: Add blogging support
We should be able to add blogs to the repo and publish them on the website.

### PR DESCRIPTION
**Pull request by simondel**
Implements #16

### CODE REVIEW

**Comment by simondel** [comment]
TODO:
* Sort blogs by date (desc)
* Add a proper layout
* Show the date in the current culture
* Refactor example blog for future reference

**Comment by simondel** [comment]
@nicojs @philippw I've written the blog for the 0.5.5 release of Stryker. Could you two review it?

File: src/blogs/2016-12-30/stryker-0-5-5.pug [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                            'src/**/*.js' // <-- mark files for mutation here
+                        ]
+                    });
+                }
+                ```
+
+                As you can see, there is no need to specify which files Stryker should use! You only have to specify which files you want to mutate. 
+                A more detailed explanation of this feature can be found in the [stryker-karma-runner README.md](https://github.com/stryker-mutator/stryker-karma-runner/blob/master/README.md#configuring).
+
+                #### We hope you enjoy this new version of Stryker! We'd love to hear your feedback!
```
Comment (philippw) [inline src/blogs/2016-12-30/stryker-0-5-5.pug]: #### We hope you enjoy this new version of Stryker! We'd love to hear your feedback on [Gitter](https://gitter.im/stryker-mutator/stryker) and/or Twitter (use #strykermutator)!

**Pull request merged**